cmd/ebpf/objects.go

```diff
@@ -172,17 +140,7 @@ func ResizeEbpfMap(eMap **ebpf.Map, eProg *ebpf.Program, newSize uint32) error {
 	return nil
 }
 
-func (bpfObjects *BpfObjects) ResizeAllMaps(qerMapSize uint32, farMapSize uint32, pdrMapSize uint32) error {
-	// QEQ
-	if err := ResizeEbpfMap(&bpfObjects.QerMap, bpfObjects.UpfQerProgramFunc, qerMapSize); err != nil {
```

Comment:
I suppose resizing of far and qer should stay here but need to be fixed

Reply:
Should something be done with FarIdTracker and QerIdTracker too? Their map size is being set only on initialization

Reply:
Resizing works correctly on initialization only